vars/transportRequestGetTransports.groovy

```diff
@@ -0,0 +1,64 @@
+import com.sap.piper.GitUtils
```

Comment:
Not fully sure about the use case. Imagine we have several open transport requests. During a pipeline run there is nobody who can pick the appropriate transport request from a set of transport requests.

Most probably it is a valid use case to operate based on an already existing transport request. Let's assume this transport request is not provided e.g. via commit history. In this case it is required that the (open) transport request is unique.

Would be possible to fail in case of several open pull requests.

I see the functionality of retrieving the open pull request from the back-end as an auxiliary method in a CMUtils class rather than having a dedicated step for that.

Reply:
So far, the job fails in case of several open pull requests, but the logic is in the pipeline. So far, I think that it is ok like this. But we can add more functionalities with more PRs, I agree.